### ISSUE TITLE
[a11y]: Improve screen reader navigation for course sequence tabs

### ISSUE THREAD

**Issue by peterkulko** [openedx/frontend-app-learning#1881]
## Description

The course sequence tabs do not provide sufficient information to enable a person using a screen reader to effectively navigate course content. The buttons and their content are not associated and are not adjacent in the reading order of the page. A person using a screen reader would have significant difficulty understanding the top bar and tab content as currently implemented. They could easily become confused regarding which tab content they are reviewing and might miss specific course content. 

PR: https://github.com/openedx/frontend-app-learning/pull/1820
